### ISSUE TITLE
Accessibility: keyboard navigation in mixed charts cannot reach all series

### ISSUE THREAD

**Issue by pan-kot** [highcharts/highcharts#23064]
#### Expected behaviour

In charts with column and line series all series and points must be accessible with keyboard in predictable way. 

#### Actual behaviour

When iterating series the focus sometimes ignores some series or moves in an unpredictable way.

#### Live demo with steps to reproduce

https://jsfiddle.net/Lr23a8yq/3/

Steps:
1. Move focus to the chart;
2. Try navigate between series: usually, only one of the line series and column series is navigable (depends on the focus position in the chart).

https://github.com/user-attachments/assets/6e711d54-f48b-42bd-9dce-c3d8c3fc5d67

Similar issues:
1. https://github.com/highcharts/highcharts/issues/22948
2. https://github.com/highcharts/highcharts/issues/23063

#### Product version

Highcharts v12

#### Affected browser(s)

Chrome

**Comment by MarkusBarstad** [comment]
Thank you for reporting this bug, @pan-kot !

I can see that you are indeed correct.

_internal note:_
seems to have always been broken, tested as far back as `v6.0.0`
